src/test/java/com/salesforce/storm/spout/sideline/handler/SidelineSpoutHandlerTest.java

```diff
@@ -440,7 +434,7 @@ public void testLoadSidelines() {
         final Map<String, Object> config = getConfig();
 
         final DynamicSpout spout = new DynamicSpout(config);
-        spout.open(null, null, null);
+        spout.open(config, new MockTopologyContext(), null);
```

Comment:
Wrong config. This is the topology config. Probably better to pass an empty HashMap if you don’t want to do null.